comment/tests/test_utils.py

```diff
@@ -337,36 +336,34 @@ def test_for_drf(self):
         self.assertEqual(EmailInfo.CONFIRMATION_SENT, response)
 
 
-class UtilsTest(TestCase):
+class TestUtils:
```

Comment:
Why `TestCase` is removed from this test only? I would prefer using the `self.assert*` rather than python assert keyword.

Reply:
> Why TestCase is removed from this test only?

this is a luxury that you can use with `pytest`. You don't need to pass `unittest`'s `TestCase` or even django's `TestCase` when using `pytest-django`, it does it for you. You can read more about it in their documentation https://docs.pytest.org/en/latest/.

There is a bit of magic, it doesn't necessarily use `python's` `assert` statement. It automatically does some magic behind the scenes and does all the comparison for you so that you don't have to remember and write different statements(`self.assertEqual`, `self.assertTrue`) for different use-cases.

> I would prefer using the self.assert* rather than python assert keyword.

If that is the case, there is very little point in switching to the `pytest`. Feel free to close this one then.

Reply:
I see that the form of assertion `self.assertEqual` and `self.assertTrue`  is more explicit on the type of the value that's why I would prefer it.
Moreover, I wouldn't prefer to stick with one testing framework.

You have already done the work for #112, if you like you can just revert pytest changes and we can still merge this PR.

Reply:
The current change to `pytest` was made after the issue was discussed. Reverting things because one of us doesn't like the change, seems a lot of waste of time for both of us. This also makes the process more irritating than enjoyable for me.

I think we need to build consensus and be more clear before any code is written so that both of us doesn't waste time.

Reply:
First of all I would appreciate you contribution to the project.
There should not be an issue if we try something and revert it after finding out that it doesn't fit our need.
In regard to the pytest, I have told you that I never used this framework and didn't have time to fiddle with it before you implement the changes so I am sorry to waste your time.

Lastly, I think it is a good time to create some labels to tag issues that are ready for implementation to avoid wasting people's time :)